frontend/tests/controllers/ContestDetailsTest.php

```diff
@@ -1197,4 +1197,52 @@ public function testContestPrintDetails() {
         $aliases = array_map(fn ($problem) => $problem['alias'], $problems);
         $this->assertSame($aliases, $expectedAliases);
     }
+
+    /**
+     * Check that support team members can access contest details without
+     * recording first access time
+     */
+    public function testSupportTeamMemberCanAccessContest() {
+        // Get a contest
+        $contestData = \OmegaUp\Test\Factories\Contest::createContest();
+
+        // Get some problems into the contest
+        $problems = \OmegaUp\Test\Factories\Contest::insertProblemsInContest(
+            $contestData
+        );
+
+        // Create a support team member
+        ['identity' => $supportTeamMember] = \OmegaUp\Test\Factories\User::createSupportUser();
+
+        // Assert the log is empty
+        $this->assertSame(0, count(\OmegaUp\DAO\ProblemsetAccessLog::getByProblemsetIdentityId(
+            $contestData['contest']->problemset_id,
+            $supportTeamMember->identity_id
+        )));
```

Comment:
What do you think about using the assertEmpty function, which is closer to what is described in the comment:
```suggestion
        $this->assertEmpty(\OmegaUp\DAO\ProblemsetAccessLog::getByProblemsetIdentityId(
            $contestData['contest']->problemset_id,
            $supportTeamMember->identity_id
        ));
```
?

Reply:
You are right, it is a more elegant way to express the same test logic. We should update the test to use this approach instead.